docs/configuration.md

```diff
@@ -1,12 +1,38 @@
-To configure the Fractal Server one must define some environment variables.
+To configure the Fractal Server one must define some configuration variables.
+
 Some of them are required, and the server will not start unless they are set.
 Some are optional and sensible defaults are provided.
 
-> The required variables are the following
->
-> ```
-> JWT_SECRET_KEY
-> POSTGRES_DB
-> ```
+## How to set configuration
+
+There are two possibilities for setting the configuration variables:
+
+- defining them as *environment variables* in the same environment as Fractal Server
+    ```sh
+    export VAR1=value1
+    export VAR2=value2
+    fractalctl start
+    ```
+- write them inside a file called `.fractal_server.env`, located in your current working directory
+    ```sh
+    echo "VAR1=value1
+    VAR2=value2" > .fractal_server.env
```

Comment:
Let's make
```
    echo "VAR1=value1
    VAR2=value2" > .fractal_server.env
```
a bit more readable by just listing the file contents (without `echo` + `>`)